locales/en.json

```diff
@@ -1,7 +1,29 @@
 {
+  "ErrorIndividualPackagesUnsupported": "Error: In manifest mode, `vcpkg install` does not support individual package arguments.\nTo install additional packages, edit vcpkg.json and then run `vcpkg install` without any package arguments.",
+  "ErrorInvalidClassicModeOption": "Error: The option {value} is not supported in classic mode and no manifest was found.",
+  "ErrorInvalidManifestModeOption": "Error: The option {value} is not supported in manifest mode.",
+  "ErrorNoVSInstance": "Error: in triplet {triplet}: Unable to find a valid Visual Studio instance",
```

Comment:
The one before this end in `.`. I think we should either always end with `.` or always not, with a mild preference for keeping the `.`.

Reply:
(Alternately, if this is forming a block with the 'no VS instance at' it probably deserves a `:`)